### PR TITLE
Update/html block description

### LINKED ISSUE #2831: Add block description/help text to Custom HTML
All blocks should have descriptions (see #2023). The Custom HTML block is missing one.

### PR DESCRIPTION
**Pull request by DevinWalker**
## Description
Added a description for the "Custom HTML" block 

This resolves #2831

## How Has This Been Tested?
- Manually tested

## Types of changes
- UX enhancement

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by jasmussen** [comment]
Love it, thank you for PR. These changes look good to me. 👍 👍

File: blocks/library/html/index.js [hunk 1]
```diff
@@ -83,6 +85,13 @@ registerBlockType( 'core/html', {
 							onChange={ ( event ) => setAttributes( { content: event.target.value } ) }
 						/>
 					}
+					{ focus &&
+					<InspectorControls key="inspector">
```
Comment (youknowriad) [inline blocks/library/html/index.js]: Maybe just one more tab to fix the indentation.

Comment (DevinWalker) [reply blocks/library/html/index.js]: Hey @youknowriad - good catch. I just added a tab for better code clarity.

**Review by gziolo — APPROVED** [review]
All good now, thanks for fixing 👍

**Pull request merged**